source/library/com/restfb/types/Post.java

```diff
@@ -21,24 +21,23 @@
  */
 package com.restfb.types;
 
-import static com.restfb.json.JsonObject.getNames;
-import static com.restfb.util.DateUtils.toDateFromLongFormat;
-import static java.util.Collections.unmodifiableList;
-
 import com.restfb.Facebook;
 import com.restfb.JsonMapper;
 import com.restfb.JsonMapper.JsonMappingCompleted;
 import com.restfb.exception.FacebookJsonMappingException;
 import com.restfb.json.JsonObject;
 import com.restfb.types.Checkin.Place.Location;
+import com.restfb.util.DateUtils;
+import lombok.Getter;
+import lombok.Setter;
 
 import java.util.ArrayList;
 import java.util.Date;
 import java.util.List;
 
-import com.restfb.util.DateUtils;
-import lombok.Getter;
-import lombok.Setter;
+import static com.restfb.json.JsonObject.getNames;
+import static com.restfb.util.DateUtils.toDateFromLongFormat;
+import static java.util.Collections.unmodifiableList;
```

Comment:
sorry... intellij did this.  Can easily put these back in the order they were if you prefer.